api/lib/mno_enterprise/concerns/controllers/jpi/v1/marketplace_controller.rb

```diff
@@ -35,6 +35,16 @@ def index
         apps
       end
 
+    query = MnoEnterprise::Product.includes(:'values.field', :assets, :categories, :product_pricings, :product_contracts).where(active: true)
+
+    # Ensure prices include organization-specific markups/discounts
+    if params[:organization_id] && parent_organization_id
```

Comment:
No need to check `params[:organization_id]` it's already checked in `parent_organization_id`